### DISCUSSION TITLE
Transparent repeater keeping clients in same network as main router

### DISCUSSION THREAD

**Discussion by g4nd4lf** [dchristl/esp32_nat_router_extended#114] [Q&A]
Can I use this tool to create a transparent repeater by keeping clients in same network as main router.
I used this tool as repeater of my main wifi network (let say 192.100.1.x), but clients connect to the esp32 with IP 192.168.4.x. To remotely access my clients from the main network I have to manually configure port forwarding in the esp32. Can I avoid that and configure esp32 to let the main router assigns the IPs of all the clients in that range (192.100.1.x) using DHCP?
I mean, what I need is for all clients to be on the same network and on the same set of reachable IP addresses, regardless of whether they are connected to the main router or to the repeater.

**Comment by dchristl** [comment]
Hello,


I'm sorry, that's not possible. Unfortunately, the framework and hardware doesn't support it.

**Comment by 59marcopolo** [comment]
Arf, vraiment dommage de ne pas pouvoir le mettre en mode Repeater. 
Y'a vraiment pas de solution ?
